tests/models/wko/src/example.py

```diff
@@ -201,6 +201,9 @@ def path_goals(self):
         for s in self.energy_system_components["heat_source"]:
             goals.append(MinimizeSourcesHeatGoal(s))
 
+        # for s in self.energy_system_components["airco"]:
+        #     goals.append(MinimizeSourcesColdGoal(s))
+
```

Comment:
Is this still required for something, then add comment, else remove.

Reply:
Deleted